src/core/js/logging.js

```diff
@@ -0,0 +1,78 @@
+define(['core/js/adapt'], function(Adapt) {
```

Comment:
could you put some whitespace into this file?

should be spaces 4 indentation, line breaks between function definitions etc